eth/downloader/downloader.go

```diff
@@ -951,6 +956,11 @@ func (d *Downloader) fetchNodeData() error {
 				syncStatsStateDone := d.syncStatsStateDone // Thread safe copy for the log below
 				d.syncStatsLock.Unlock()
 
+				// If real database progress was made, reset any fast-sync pivot failure
+				if progressed && atomic.LoadUint32(&d.fsPivotFails) > 1 {
```

Comment:
That's not really a problem. I just want to check whether I can reset it to 1 or not. If it gets reset twice by accident so be it, no biggie.